app/models/micropost.rb

```diff
@@ -0,0 +1,19 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  has_one_attached :image
+
+  scope :order_created_at, ->{order created_at: :desc}
```

Comment:
scope là gì ?

Reply:
scope giúp định nghĩa các điều kiện truy vấn,  có thể kết nối nhiều scope với nhau mà không tạo ra nhiều câu truy vấn. Về bản chất thì scope là 1 class method (có thể gọi là class method động).Có thể tái sử dụng lại scope, giúp code tường minh hơn, dễ hiểu dễ bảo trì sửa đổi hơn, có thể kết hợp được với toán tử điều kiện và luôn đảm bảo method chain